Makefile

```diff
@@ -68,25 +69,25 @@ libcamlzip.a: $(C_OBJS)
 
 clean:
 	rm -f *.cm*
-	rm -f *.o *.a *.so
+	rm -f *.o *$(EXT_LIB) *$(EXT_DLL)
 
 install:
 	mkdir -p $(INSTALLDIR)
```

Comment:
If we make ocamlfind mandatory, this legacy, ocamlfind-free "install" procedure should be removed.